Readme.MD

```diff
@@ -58,6 +58,7 @@ import {
 - [x] [Grid Component](https://github.com/react-native-community/react-native-elements#grid-component)
 - [x] [Slider Component](https://github.com/react-native-community/react-native-elements#slider-component)
 - [x] [Tile Component](https://github.com/react-native-community/react-native-elements#tile-component)
+- [x] [Avatar Component](https://github.com/react-native-community/react-native-elements#avatar-component)
 
 ## Roadmap
```

Comment:
This should be updated. Ideally it should have all our roadmap features with [FIRST CONSTRUCTION] tag on the relevant ones.